db_management/db_upgrade.rst

```diff
@@ -0,0 +1,422 @@
+Sales, Purchase, and Manufacturing.
+
+So, in a nutshell, what can impact your upgrade lead time?
+
+* Source & targeted versions
+* Installed apps
+* Volume of data
+* Amount of customization (models, fields, methods, workflows, reports, website, etc.)
+* Installation of new apps or configuration changes after the start of the test phase
+
+Usually, the delays experienced during the first request (waiting time between the time you
+submitted your first request for a test upgrade) can generally give you an idea of the time to wait
+for the production request.
+
+.. _db-upgrade/custom-modules:
+
+Upgrade of the custom modules
+-----------------------------
+
+As stated in our :ref:`Odoo Enterprise Subscription Agreement, section "5.1 Standard charges"
+<charges_standard>`, this optional service is subject to additional fees.
+
+If you have a custom code, you can choose to have it upgraded by our services, by one of our
+`partners <https://www.odoo.com/partners>`_ or you can do it yourself.
+
+.. note::
+   If you need Odoo assistance on this matter, please contact your Odoo Account Manager or contact
+   our `Sales department <mailto:sales@odoo.com>`_.
+
+.. _db-upgrade/editions-change:
+
+Editions change (from Community to Enterprise)
+----------------------------------------------
+
+An upgrade does not cover a change of `Editions <https://www.odoo.com/page/editions>`_
+
+.. note::
+   If you need Odoo assistance on this matter, please contact your Odoo Account Manager or contact
+   our `Sales department <mailto:sales@odoo.com>`_.
+
+.. _db-upgrade/hosting-types-switch:
+
+Switching the hosting types (Self-hosted vs Online vs Odoo.sh)
+--------------------------------------------------------------
+
+An upgrade does not cover a change of `Hosting types <https://www.odoo.com/page/hosting-types>`_
+
+Open the following link to get :doc:`more information about how to change your hosting type
+<hosting_changes>`.
+
+.. note::
+   If you need Odoo assistance on this matter, please contact your Odoo Account Manager or contact
+   our `Sales department <mailto:sales@odoo.com>`_.
+
+.. _db-upgrade/release-notes:
+
+Release Notes by version
+------------------------
+
+Open our `Release Note <https://www.odoo.com/page/release-notes>`_ page to get a summary of the new
+features and improvements made in each version.
+
+.. _db-upgrade/assistance:
+
+Assistance
+==========
+
+.. _db-upgrade/contact:
+
+Contact our Upgrade service support
+-----------------------------------
+
+Should you have any more questions about the upgrade, do not hesitate to send a message to `Odoo
+Upgrade Team <mailto:upgrade@odoo.com>`_. We will be very pleased to answer it as soon as possible.
+
+.. _db-upgrade/supported-versions:
+
+Supported versions
+------------------
+
+Please note that Odoo provides support and bug fixing only for the three last major versions of Odoo.
+
+This is a factor to take into consideration before upgrading. If you are in an older version, we
```

Comment:
```suggestion
This is a factor to take into consideration before upgrading. If you are on an older version, we
```
Honestly don't know which one is correct. I'm highlighting it just in case.